02_bubble/undrewb/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+/*
+	https://github.com/undrewb/go-course/tree/lab1/02_bubble
```

Comment:
Seems like the wrong path (lab1/02_bubble)

Reply:
fixed.